hoodie-client/src/main/java/com/uber/hoodie/config/HoodieCompactionConfig.java

```diff
@@ -102,6 +101,10 @@
   public static final String DEFAULT_PAYLOAD_CLASS = HoodieAvroPayload.class.getName();
   public static final String PAYLOAD_CLASS = "hoodie.compaction.payload.class";
 
+  public static final String MAX_SIZE_IN_MEMORY_PER_COMPACTION_IN_BYTES_PROP = "hoodie.compaction.spill.threshold";
+  // Default memory size per compaction, excess spills to disk
+  public static final String DEFAULT_MAX_SIZE_IN_MEMORY_PER_COMPACTION_IN_BYTES = String.valueOf(1024*1024*1024L); //1GB
```

Comment:
@vinothchandar here

Reply:
you mean usage inside compaction vs cow/mergehandle? In that case, we are doing one of those cases right.

Reply:
Yeah, we are doing one of those cases but seems like `DEFAULT_MAX_SIZE_IN_MEMORY_PER_COMPACTION_IN_BYTES` name to set the size of spillable map is weird for cow ?

Reply:
@vinothchandar ^

Reply:
answer would depend on if the SpillableMap is ever used by COW path..I don't think thats true today.. So its okay for now. We can add a new param when we change COW actually.. Thats my take.

Reply:
same here